scripts/load_test.py

```diff
@@ -392,7 +392,7 @@ def buildCoros(coroFunc, corosArgs):
     clientPoll = ClientPoll(args.clientsListFilePath,
                             args.numberOfClients, args.numberOfClientsToSkip)
 
-    with Looper(debug=True) as looper:
+    with Looper() as looper:
```

Comment:
Why config.LOOPER_DEBUG is not used here?

Reply:
Because we decided to use default debug value of Looper class (False) or use False explicitly for testing functionality. As I understand, 'scripts/load_test.py' script is related to tests.